ff/src/biginteger/mod.rs

```diff
@@ -95,7 +95,9 @@ impl<const N: usize> BigInt<N> {
     }
 
     pub const fn one() -> Self {
-        BigInt!("1")
+        let mut one = Self::zero();
```

Comment:
I guess it also makes it easier to be a `const fn`. The reason for `zero()` is to avoid circular calls as BigInt! calls `zero()`, but this is not strictly needed for `one()`. 

That said, this one is better.

Reply:
It is a bit more performant for compile time, I think.